db/migrate/20160317194215_remove_miq_user_role_from_miq_groups.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveMiqUserRoleFromMiqGroups < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :miq_groups, :miq_user_role_id, :string
```

Comment:
HAH. Rails generator magic fail (on my part). Good catch.

On Thursday, March 17, 2016, Joe Rafaniello <notifications@github.com>
wrote:

> In db/migrate/20160317194215_remove_miq_user_role_from_miq_groups.rb
> <https://github.com/ManageIQ/manageiq/pull/7302#discussion_r56581361>:
>
> > @@ -0,0 +1,5 @@
> > +class RemoveMiqUserRoleFromMiqGroups < ActiveRecord::Migration[5.0]
> > +  def change
> > +    remove_column :miq_groups, :miq_user_role_id, :string
>
> is this a string? [image: :confused:]
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/7302/files/0e05d3294527d5fec98ea5435155d45a08e1ed77#r56581361>
>


-- 
Chris Arcand
@chrisarcand